ClientSecurity/Utilizing_SLACK_for_IPS_Event_Notifications.md

```diff
@@ -0,0 +1,76 @@
+# Utilizing SLACK for IPS Event Notifications 
+
+### Overview
+
+Slack is a team collaboration tool that offers persistent chat rooms organized by topic and also has private groups and direct messaging capabilities. Slack is available as long as you want and can be utilized with an unlimited number of people.
+
+Our Intrusion Prevention System (IPS) provides real-time event notifications that can be integrated with Slack messaging. Using the Slack integration for “Incoming WebHooks”, our IPS can be set up to send event details on the rule triggered, source and destination information to a Slack channel so it can be quickly reviewed by your team or archived for review at a later time.
+
+### Signing Up for Slack
+
+Utilize the link below to sign up and get started with Slack!
```

Comment:
update text links to say something simple that avoid printing out the full URL in the kb.

```
* [Getting Started with Slack](//slack.zendesk.com/hc/en-us/articles/206480347-Getting-started-with-Slack)
```